src/App.js

```diff
@@ -1,16 +1,25 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./Clock.js"
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import {WorldClock, clockData} from './worldclock.js'
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <Container>
+            <Row>
+              <Col>City</Col>
+              <Col>Time</Col>
+            </Row>
+            <WorldClock clockData={clockData}/>
```

Comment:
I find it a bit odd that you are importing clockData from WorldClock, to then pass it as props to WordClock component :D!